charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml

```diff
@@ -238,17 +238,18 @@ data:
         Rename              msg    log
         Rename              logger source
 
-    # Kubernetes filter
-    [FILTER]
-        Name                kubernetes
-        Match               kubernetes.*
-        Kube_Tag_Prefix     kubernetes.var.log.containers.
-        Kube_URL            https://kubernetes.default.svc.cluster.local:443
-        Buffer_Size         1M
-        Labels              On
-        Annotations         On
-        tls.verify          Off
-        K8S-Logging.Exclude Off
+    # Use this filter to retrieve metadata from the Apiserver
+    # # Kubernetes filter
+    # [FILTER]
+    #     Name                kubernetes
+    #     Match               kubernetes.*
+    #     Kube_Tag_Prefix     kubernetes.var.log.containers.
+    #     Kube_URL            https://kubernetes.default.svc.cluster.local:443
+    #     Buffer_Size         1M
+    #     Labels              On
+    #     Annotations         On
+    #     tls.verify          Off
+    #     K8S-Logging.Exclude Off
```

Comment:
Why is it commented?

Reply:
If we decide in the future to bring it back :D